### DISCUSSION TITLE
OpenId Questions

### DISCUSSION THREAD

**Discussion by shwagwsu** [drakkan/sftpgo#828] [Q&A] [answered]
I'm using version 2.2.3.1 and trying configure OpenId on both Linux.  I have followed below article when configuring keycloak and sftpgo

https://github.com/drakkan/sftpgo/blob/main/docs/oidc.md

I have the following in my configuration 

 "oidc": {
         "client_id": "sftpgo-client",
         "client_secret": "jRsmE0SWnuZjP7djBqNq0mrf8QN77j2c",
         "config_url": "http://localhost:8080/realms/psft-server",
         "redirect_base_url": "http://localhost:8081",
         "username_field": "preferred_username",
         "role_field": "sftpgo_role",
         "custom_fields": []
        }

I could not see the "login with openid connect" button on sftpgo login page.  I have also tried using incorrect data and still no button and not seeing the values actually being picked up.  I have tried the same on windows with the same result.

Could you please help me?  Is there a certain version or variant I need to use?

**Comment by drakkan** [comment, marked as answer]
Hi,

for OIDC support you need to use the development version, it is not yet included in any "stable" release. It will be included in v2.3.0 once I'll release it

**Reply by shwagwsu** [reply]
Thanks..
